backend/api/visualization.proto

```diff
@@ -58,7 +58,7 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
 service VisualizationService {
   rpc CreateVisualization(CreateVisualizationRequest) returns (Visualization) {
     option (google.api.http) = {
-      post: "/apis/v1beta1/visualizations"
+      post: "/apis/v1beta1/visualizations/{namespace}"
```

Comment:
I expect this to be backward compatible (will see if tests complain).
If it's not, then plan B is to include it in the body as `body: "*"`

Reply:
I am not sure We have test coverage, but the only client is UI. I can update UI usage before next release.
the only requirement is, namespace should be optional

Reply:
One reason I put namespace in the end is that I assume if namespace is empty, it would result in the same endpoint as before. 
Here is the api-server I built from this change: gcr.io/chesu-dev/api-server:pr3588
Can you help verify that when you make the UI change?

I feel like having namespace in the path makes more sense, so want to try this first. But there're other options:
1. use `body: "*"`
2. move `namespace` into the `visualization` message.

Reply:
Yep, if this works, I think that would be the best, otherwise, 1. sounds good to me

Reply:
I'm testing it

Reply:
@chensun it seems this is a backward incompatible change. Frontend requests `/apis/v1beta1/visualizations` and got `Not Found` error.

Reply:
UPDATE: I edited local client and requested `/apis/v1beta1/visualizations/` instead, now it works properly.

I think we can merge this then. namespace is optional

Reply:
okay, thanks for testing. I changed it to using `body: "*"`. Currently building a new image, which should be available in a few minutes:
gcr.io/chesu-dev/api-server:pr3588v2

Reply:
Hmm, I saw you already updated. I think this is also good...
You can decide on this.

Reply:
Sorry, didn't see your latest comments until I pushed new change.
Question: is the last `/` in `/apis/v1beta1/visualizations/` required for it to work? 
I personally prefer namespace in the path rather than in the body, but I don't feel too strongly about it. How does this affect UI implementation? which way do you prefer?

Reply:
Will stick with `post: "/apis/v1beta1/visualizations/{namespace}"`